www/app/app.ts

```diff
@@ -95,6 +96,7 @@ export class App {
     this.eventQueue.startPublishing();
 
     this.displayZeroStateUi();
+    this.maybeShowPrivacyView();
```

Comment:
I find that `maybe` methods makes the code harder to understand.

It's usually more readable if you separate the condition from the action.
You could change this code to read something like:
```javascript
if (!this.isPrivacyTermsAccepted()) {
  this.showPrivacyTerms();
}
```

even though it's longer, I don't need to look inside the method anymore to know what the condition is.

Reply:
Done in a2ecc48.